src/lib/db/repositories/types.ts

```diff
@@ -0,0 +1,3 @@
+export type RepoResponse<T = void> =
```

Comment:
opinionated nit: let's not abbreviate

```suggestion
export type RepositoryResponse<T = void> =
```